client/service/src/client/wasm_overwrite.rs

```diff
@@ -15,6 +15,27 @@
 
 // You should have received a copy of the GNU General Public License
 // along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+//! # WASM Local Blob-Overwrite
+//!
+//! WASM Local blob overwrite provides tools to replace on-chain WASM with locally-built WASM.
```

Comment:
This "locally-built" wasm sounds weird. The wasm could be build everywhere and still be usable here. 
```suggestion
//! WASM Local blob overwrite provides tools to replace on-chain WASM with a local WASM blob.
```

Maybe something like this? And the other ones should be changed based on context. Not always "local" makes sense.

Reply:
The distinction is between blobs on chain and any other blobs, but `offchain WASM blob`, while technically accurate, is just confusing. I guess we could go with `custom`? If they didn't make any change to it there's no sens overriding.

Reply:
I think custom should get across the right message, I think I'll go with that

Reply:
Yeah sounds good :) 

I also thought about offchain and came to the same conclusion, that it is wrong here ;)